packages/discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -129,7 +134,7 @@ def update_track_price_history(
             if "splits" in usdc_purchase:
                 splits = usdc_purchase["splits"]
                 # TODO: [PAY-2553] better validation of splits
-                if isinstance(splits, dict):
+                if isinstance(splits, list):
```

Comment:
If I understand correctly, this is going to break uploads in clients that haven't updated to use the correct format?
If so, I'm concerned that the user would succeed in uploading all the content and then we would fail to index the upload here.

Reply:
Yes but this is also going out at the same time as the migration to convert those, and the entity manager changes to convert on writes